<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="eyeGroupBox">
        <property name="title">
         <string>VR eye</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>50</width>
           <height>42</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QRadioButton" name="rightEyeRadioButton">
            <property name="text">
             <string>Right</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">eyeButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="leftEyeRadioButton">
            <property name="text">
             <string>Left</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">eyeButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Codec</string>
        </property>
        <widget class="QListWidget" name="codecListWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>161</width>
           <height>171</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Width</string>
          </property>
          <property name="buddy">
           <cstring>videoWidthSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Height</string>
          </property>
          <property name="buddy">
           <cstring>videoHeightSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="videoWidthSpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Bitrate</string>
          </property>
          <property name="buddy">
           <cstring>videoBitrateSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>kb/s</string>
          </property>
          <property name="buddy">
           <cstring>videoBitrateSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="videoBitrateSpinBox">
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="videoHeightSpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>FrameRate</string>
          </property>
          <property name="buddy">
           <cstring>videoFramerateSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="videoFramerateSpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Serial Port</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Stop Bits</string>
        </property>
        <property name="buddy">
         <cstring>portStopBitsComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="portStopBitsComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="portRateSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>9600</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="portDataBitsComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rate</string>
        </property>
        <property name="buddy">
         <cstring>portRateSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Data Bits</string>
        </property>
        <property name="buddy">
         <cstring>portDataBitsComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>portLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="portParityComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="portLineEdit">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>COM2</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Parity</string>
        </property>
        <property name="buddy">
         <cstring>portParityComboBox</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="eyeButtonGroup"/>
 </buttongroups>
</ui>
